stable/voyager/templates/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
There is already a file called svc.yaml that does this. You probably need to either update that one or use a git mv .

Reply:
Fixed now.